### DISCUSSION TITLE
After updating to version 8.12.0 gotenberg no longer starts

### DISCUSSION THREAD

**Discussion by hatty24** [gotenberg/gotenberg#1026] [Q&A]
Hello everyone,

I use gotenberg with paperless-ngx in a docker container.

After the update to 8.12.0 gotenberg no longer starts.

The following error message appears:
[FATAL] provision module api: get routers: provision module chromium: get PDF engine provider: get module: provision module pdfengines: get PDF engines: provision module pdfcpu: PDFCPU_BIN_PATH environment variable is not set

Where do I have to set this variable and what should it be?

Everything worked fine before the update.

I would be very grateful for some help!

**Comment by gulien** [comment]
That’s weird 🤔 it’s defined here in the image:

https://github.com/gotenberg/gotenberg/blob/8ff9d3bcf113a938fcfb16eb3838e107360c1014/build/Dockerfile#L228

**Reply by gulien** [reply]
Are you using a particular variant?

**Reply by hatty24** [reply]
I have updated it via the Container Manager in Synology DSM. This has always worked until now.

**Comment by hatty24** [comment]
@gulien 
The entry was missing.
I have now added it manually and it's working again!

Many thanks for the quick help.

**Reply by gulien** [reply]
I don’t get it, the env var should always be present 😅 unless Synology DNS has its own build system?

**Reply by hatty24** [reply]
I also don't understand. The container manager in DSM also gets its updates from docker hub.
Anyway, at least I have it running again. 😊

**Comment by gregkonline** [comment]
Had the same issue. A rebuild is required. I rebuilt the project in the Synology Container Manager, and all works well now.
